com.github.reduz.ChibiTracker.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.github.reduz.ChibiTracker
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: chibitracker
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
Why it does require host filesystem access?

Reply:
I will change it to home access. The idea was letting the app access mounted drives, but overall /home only should be a better idea.